library/include/modules/EventManager.h

```diff
@@ -20,6 +20,7 @@ namespace DFHack {
             enum EventType {
                 TICK,
                 JOB_INITIATED,
+                JOB_STARTED, //has a worker
```

Comment:
New event types also need to be added to `plugins/lua/eventful.lua`.

Reply:
I added the event to that file and filled in the blanks with `plugins/eventful.cpp`
I haven't tested that it works, but the integration looks to be complete in the same way the other events have been integrated.